dart-runtime/lib/types.dart

```diff
@@ -124,6 +124,12 @@ encode(Map<String, Object> typeTable, String path, Object type, Object value) {
               .toUtc()
               .toIso8601String()
               .replaceAll("Z", "");
+        case "void":
+          if (!(value != null)) {
+            throw SdkgenTypeException(
+                "Invalid Type at '$path', expected ${jsonEncode(type)}, got ${jsonEncode(value)}");
+          }
```

Comment:
This check is causing a breaking change with other clients, as it doesn't allow an API to change the return type of a method from void to any type. It should be removed :)

Reply:
ok, fixed!